src/System.Globalization.Calendars/tests/Misc/MiscCalendars.cs

```diff
@@ -39,35 +37,11 @@ public static void HijriTest()
         }
 
         [Fact]
-        public static void GregorianTest()
-        {
-            GregorianCalendar gCal = new GregorianCalendar();
-            DateTime dTest = gCal.ToDateTime(1600, 1, 1, 0, 0, 0, 0);
-            Assert.Equal(dTest, new DateTime(1600, 1, 1));
-        }
-
-        [Fact]
         public static void JapaneseTest()
         {
             JapaneseCalendar jCal = new JapaneseCalendar();
             DateTime dTest = jCal.ToDateTime(1, 1, 8, 0, 0, 0, 0);
             Assert.Equal(dTest, new DateTime(1989, 1, 8));
         }
-
-        [Fact]
-        public static void KoreanTest()
-        {
-            KoreanCalendar jCal = new KoreanCalendar();
-            DateTime dTest = jCal.ToDateTime(3933, 1, 1, 0, 0, 0, 0);
-            Assert.Equal(dTest, new DateTime(1600, 1, 1));
```

Comment:
Can you point me to what this test is duplicating?  I don't see this 3933 number anywhere else in the project, other than in a PersianCalendar test.